### PR TITLE
auth0 implementation

### PR DESCRIPTION
**Pull request by beeena**
Here In this PR AuthO is Implemented.

And continue working with custom login/signup Form with AuthO.

### CODE REVIEW

**Comment by cdmbase** [comment]
Can one of the admins verify this patch?

File: servers/frontend-server/src/app/Main.tsx [hunk 1]
```diff
@@ -22,6 +22,9 @@ import { ServerError } from './Error';
 import { PersistGate } from 'redux-persist/integration/react';
 import { persistStore, persistReducer } from 'redux-persist';
 
+import { BrowserRouter as Router } from "react-router-dom";
+import Auth0ProviderWithHistory from "../../../../packages-modules/counter/browser/src/autho/auth-provider-with-history";
```
Comment (veeramarni) [inline servers/frontend-server/src/app/Main.tsx:26]: it should be `import { Auth0ProviderWithHistory } from '@sample-stack/counter-moudle-browser`

**Comment by beeena** [comment]
> Can one of the admins verify this patch?

Not Yet. Just login with Google.

File: servers/frontend-server/src/app/Main.tsx (hunk 1 above)
Comment (beeena) [reply servers/frontend-server/src/app/Main.tsx:26]: Okay Sure.

**Pull request closed without merging**